### DISCUSSION TITLE
Pyrhon venv active by default

### DISCUSSION THREAD

**Discussion by mubarak-k** [netbox-community/netbox#15495] [Help Wanted!]
I have installed and configured netbox latest version, now able to test python virtual environment and access netbox home page.

However, after exit the prompt netbox page unable access. Again I have to activate venv and run python manage.py script while results the home page to accessible.

Kindly help me to resolve this issue.  How to make sure the venv activate all the time.

**Comment by jeremystretch** [comment]
It sounds like you need to read through the [installation instructions](https://docs.netbox.dev/en/stable/installation/) again, which specifically note:

> The development server is for development and testing purposes only. It is neither performant nor secure enough for production use. **Do not use it in production.**

**Reply by mubarak-k** [reply]
Thanks for your immediate response.

Kindly let me know how to setup for production.

Highly appreciated if you share the steps or link.

**Reply by jeremystretch** [reply]
Amazing.

**Reply by bitcollector1** [reply]
This is what happens when you become popular ;)
